### DISCUSSION TITLE
dimming of led ring

### DISCUSSION THREAD

**Discussion by dingo35** [serkri/SmartEVSE-3#95] [Ideas]
[1.5.2+-dimled.zip](https://github.com/serkri/SmartEVSE-3/files/10539393/1.5.2%2B-dimled.zip)

Attached the binaries of a firmware 1.5.2 + some small changes + a change that dims the ledring (hardcoded).

**Comment by nekocentral** [comment]
Ledring was still somewhat bright but a lot better yea, still lights up my complete garage when charging
